### ISSUE TITLE
Installing mint with language pack and questions

### ISSUE THREAD

**Issue by Qronikarz** [linuxmint/mint22-beta#4]
1. The current Mint installer downloads the language-packs during the install and also during the same install deletes them. In previous versions there was just a pop up after installation to download language packs in mintReport and I think it was faster this way. Now you wait for all language packs to download and then wait to delete unused ones.

2. When installing Mint you first pick the installer language and then right after it asks you to pick system language. It's good because somebody can install the OS in a different language for other people, but the installer could at least pick the previous language automatically.
Example: I picked Polish language during the setup and in the next step the the English was the default. This step should remain, but installer should pick the Polish language.
I don't remember if that was different in previous version, but if not then it's a good improvement in my opinion.

3. Is there a repository where users can suggest the changes to the installer? I found some things that I don't think are Mint 22 related:
* The first welcome screen during the installation which shows netflix uses the "included software" and lists "firefox", "netflix" and "youtube". I have never seen these apps in Mint. It should either be changed to "recommended software" just like all the other screens or specified that you can create PWA of those sites with webapp-manager
* During the Wine screen I think it would be good to recommend Bottles, Steam, Lutris and Heroic launchers since they make setting up Wine painless.

**Comment by Qronikarz** [comment]
A follow up from my Mint 21.3 SD card reinstall.

2. Mint 21.3 does in fact automatically select Polish OS language and Polish keyboard when you select the Polish language during first install step.

3. Yes, it's not a Mint 22 issue, but I feel like that's even better time for some changes to the installer
![installerlook](https://github.com/linuxmint/mint22-beta/assets/84787215/3e76d2ce-ff9c-4822-b284-ec35b36be29c)
It kind of looks out of place with that gray areas and while I personally love Firefox Quantum with it's Photon UI, it got replaced with Proton in Firefox v89 released 2021.06 (https://github.com/black7375/Firefox-UI-Fix/wiki/%5BArticle%5D-0.-Firefox-UI-UX-history)

**Comment by haggen88** [comment]
@clefebvre  A little question. Are the unnecessary fonts also removed with the language packs?. I am referring to asian, etc. fonts that I will not use.

**Comment by fredcw** [comment]
@haggen88 I think it's good to have as much of Unicode covered as possible. If you're reading some webpage about different languages or scripts, you don't want missing characters. Font files don't take up too much space (10's of MB) unless you have lots of different typefaces for each script.

**Comment by clefebvre** [comment]
A couple of things first...

- fonts-noto-core was missing in this BETA. With it added many fonts will render properly in the stable release.
- ubiquity, the installer we're using in Mint, is full of nice features (some of which we don't have in LMDE, such as OEM installations) but it's also full of technical debt (we don't like working with it, we change as little as possible in it, it's not designed the way we'd like and we're not redesigning it). Long term we'll probably add the missing features to the LMDE installer (live-installer) and port it to Mint.

**Comment by clefebvre** [comment]
> 1. The current Mint installer downloads the language-packs during the install and also during the same install deletes them. In previous versions there was just a pop up after installation to download language packs in mintReport and I think it was faster this way. Now you wait for all language packs to download and then wait to delete unused ones.

Yes. If you're online it downloads anything missing, or even updates what's already installed.  And then no matter what, further down the installation process it removes what it doesn't need. It's not ideal but that's the way ubiquity works.

>     3. Is there a repository where users can suggest the changes to the installer? I found some things that I don't think are Mint 22 related:
> 
> 
>     * The first welcome screen during the installation which shows netflix uses the "included software" and lists "firefox", "netflix" and "youtube". I have never seen these apps in Mint. It should either be changed to "recommended software" just like all the other screens or specified that you can create PWA of those sites with webapp-manager
> 
>     * During the Wine screen I think it would be good to recommend Bottles, Steam, Lutris and Heroic launchers since they make setting up Wine painless.

This is handled by ubiquity-slideshow-mint. It doesn't have a github repository but it's something we can improve. Again though, this is tied to ubiquity and long-term we'll abandon it. I'm taking note of this though.
